### PR TITLE
Ability to set multiple properties with object

### PR DESCRIPTION
**Pull request by andrewsokolov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -83,7 +83,13 @@ Configstore.prototype.get = function (key) {
 
 Configstore.prototype.set = function (key, val) {
 	var config = this.all;
-	config[key] = val;
+	if (key instanceof Object) {
```
Comment (sindresorhus) [inline index.js]: `instanceof Object` can fail in many scenarios. Better to just check `arguments.length`.

**Comment by sindresorhus** [comment]
Can you update the docs too?

**Comment by andrewsokolov** [comment]
Fixed and docs updated.

**Comment by hemanth** [comment]
:+1:

**Comment by andrewsokolov** [comment]
@hemanth Can you publish those changes to npm?

**Comment by hemanth** [comment]
@andrewsokolov, @sindresorhus must help us.

**Pull request merged**